bin/cli.js

```diff
@@ -11,6 +11,7 @@ var pkgJson = require('../package.json'),
 
 program
     .version(pkgJson.version)
+    .description(pkgJson.description + '. Returns 0 on success, 1 on errors and 2 on warnings.')
```

Comment:
Not sure we need this description.
0 & 1 is standard behaviour.

2 needs discussion as of https://github.com/TryGhost/gscan/pull/108/files#r197758659.

Reply:
Yeah, I knew of that, but decided to use it anyways.
Don't remember why. ;)

Reply:
Okay, no problem. Could you please revert the error code "2"? Thanks!

Reply:
Done.

Reply:
?

Reply:
Did i misunderstand?
I had changed it already to 42 and since you wrote "revert" I switched back from 42 to 2.

Reply:
> Not sure about the status code "2" here 🤔
> Please take a look at e.g. this link, which describes the meanings of exit codes.

Exit code 2 has a different meaning.

Reply:
What do you suggest to use instead? 0 or something else?